operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -76,12 +75,7 @@ func convertDBPersistenceToPersistence(p *platform.DBPersistence) *platform.Pers
 func getPersistenceByTarget(central *platform.Central, target PVCTarget) *platform.Persistence {
 	switch target {
 	case PVCTargetCentral:
```

Comment:
Is this still a possible case?

Reply:
Yes, we shall remove the owner for Central.